.github/workflows/test.yml

```diff
@@ -9,9 +9,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [ubuntu-20.04]
         tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8']
-    runs-on: ${{ matrix.runs-on }}
+        coveralls: [false]
+        include:
+          - tarantool: '2.8'
+            coveralls: true
```

Comment:
Why do you need both `(2.8, false)` and `(2.8 true)`?

Reply:
Fixed.